charts/uptime-kuma/values.yaml

```diff
@@ -118,3 +118,6 @@ volume:
   # storageClassName:
   # Reuse your own pre-existing PVC.
   existingClaim: {}
+
+strategy: 
+  type: Recreate
```

Comment:
We need a newline here:
https://github.com/dirsigler/uptime-kuma-helm/runs/6376138441?check_suite_focus=true#step:7:37

Reply:
small change; therefore did it myself.